server/apps/search_providers/aap_mm/service.py

```diff
@@ -63,6 +64,7 @@ def create(self, docs, **kwargs):
             dest_doc[ITEM_STATE] = doc.get(ITEM_STATE, CONTENT_STATE.FETCHED)
             dest_doc[INGEST_ID] = archived_doc[config.ID_FIELD]
             dest_doc[FAMILY_ID] = archived_doc[config.ID_FIELD]
+            dest_doc[ITEM_OPERATION] = ITEM_FETCH
```

Comment:
Do we have a behave test verifying that multimedia content fetch does not have dateline?